tests/test_doctestplus.py

```diff
@@ -510,7 +510,7 @@ def test_text_file_comments(testdir):
         '--doctest-glob', '*.rst',
         '--doctest-glob', '*.tex',
         '--doctest-glob', '*.txt'
-    ).assertoutcome(passed=3)
+    ).assertoutcome(passed=0)
```

Comment:
This is testing that the tests are commented, so they are not run, they should not "pass". Same below.